packages/datetime/src/timePicker.tsx

```diff
@@ -243,7 +260,13 @@ export class TimePicker extends React.Component<TimePickerProps, ITimePickerStat
         const onClick = () => (isDirectionUp ? this.incrementTime : this.decrementTime)(timeUnit);
         // set tabIndex=-1 to ensure a valid FocusEvent relatedTarget when focused
         return (
-            <span tabIndex={-1} className={classes} onClick={onClick}>
+            <span
+                aria-controls={this.timeInputIds[timeUnit]}
+                aria-label={`${isDirectionUp ? "Increase" : "Decrease"} ${timeUnit}`}
```

Comment:
it's good that you included the time unit here, but it looks like it will not print nicely for some of the units. can you create a helper function in `timeUnit.ts` to pretty print the time unit enum? something like:

```ts
/** Prints a descriptive label representing the plural of th given time unit. */
export function printTimeUnit(unit: TimeUnit) {
    return {
        [TimeUnit.HOUR_24]: "hours (24hr clock)",
        [TimeUnit.HOUR_12]: "hours (12hr clock)",
        [TimeUnit.MINUTE]: "minutes",
        [TimeUnit.SECOND]: "seconds",
        [TimeUnit.MS]: "milliseconds",
    }[unit];
}
```

... and then use that helper function in this label value.

Reply:
done